src/App.tsx

```diff
@@ -105,6 +109,16 @@ const App: React.FC = () => {
                         exact
                         component={VerifyEmail}
                       />
+                      <Route
+                        path={Routes.SET_CONTACTS}
+                        exact
+                        component={SetContacts}
+                      />
+                      <Route
+                        path={Routes.SIGNUP_CONFIRMATION}
+                        exact
+                        component={SignupConfirmation}
+                      />
```

Comment:
I feel like we should just create a `routes` object, mapping a Route to a component. And simply just map through its entries. Then we can just add entries to the object. @jackblanc thoughts?

Reply:
Oh that's interesting, I like that. We could also try to store a concept of like "minimum privilege level" required to access the page.

Reply:
That's a good idea! Definitely outside the scope of this PR, but could make routing cleaner going forward.